src/raps/actions/swap.ts

```diff
@@ -47,12 +47,14 @@ const swap = async (
   const { gasFeeParamsBySpeed, selectedGasFee } = store.getState().gas;
   let gasParams = parseGasParamsForTransaction(selectedGasFee);
   // if swap isn't the last action, use fast gas or custom (whatever is faster)
-  if (
-    currentRap.actions.length - 1 > index ||
-    !gasParams.maxFeePerGas ||
-    !gasParams.maxPriorityFeePerGas ||
-    !gasParams.gasPrice
-  ) {
+  const isL2 = isL2Network(
+    ethereumUtils.getNetworkFromChainId(parameters?.chainId || 1)
```

Comment:
```
import { ChainId } from '@uniswap/sdk';
const isL2 = isL2Network( ethereumUtils.getNetworkFromChainId(parameters?.chainId || ChainId.MAINNET)
```

Nit picky suggestion, but if you ever need to see where we are using mainnet chain id defaults, this is a lot easier to search for than `1`.

Reply:
oh yes nice, i'm gonna try to find a way to do the same without `@uniswap/sdk` since we don't have it anymore

Reply:
ah is the same with `@rainbow-me/swaps` 🤝

Reply:
@derHowie pushed!

Reply:
ahh good call, I got you half way there. haha.